catalogue/templates/catalogue/includes/component-card.html

```diff
@@ -38,6 +38,17 @@
             Learn More
           </a>
         </div>
+        <button
+          id="addComponent"
+          class="button is-light level-right"
+          hx-post="{% url 'catalogue:add_cart' c.id %}"
+          hx-target="#shopping_card"
+          hx-swap="innerHTML"
+        >
```

Comment:
can you move the button to the bottom right corner of the card? `level-right` is not working properly and disturbs the "Learn More" position.